lib/middleware/util.ts

```diff
@@ -54,3 +54,16 @@ export function after(req: Interface.IOurRequest,
 {
     req.log.info({res: res}, 'Response sent.');
 }
+
+export function rejectHTTP(req: Interface.IOurRequest,
+                           res: Interface.IOurResponse,
+                           next: restify.Next): void
+{
+    if (!conf.get('https.enable')) {
```

Comment:
This check seems odd -- we're not checking if the request itself is http (vs https), but rather whether the server is configured for https. Should this check req.isSecure() or some such?